rag/advanced_rag/harness/tools/registry.py

```
@@ -0,0 +1,141 @@
+            "parameters": {
+                "type": "object",
+                "properties": {
+                    "topic": {"type": "string", "description": "要定位的主题"},
+                },
+                "required": ["topic"],
+            },
+        },
+    }
+
+
+def _inspector_schema(name: str, desc: str, props: dict = None) -> dict:
```

Ruff (0.15.20)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)